### PR TITLE
ab: left menu navigation arrow

### LINKED ISSUE #390: Add arrow indicator in left nav to orient user
As an instructor, it's hard to know where I am in the Assignment Builder. An indicator in the left nav pointing to the current screen would be helpful.

### PR DESCRIPTION
**Pull request by annagav**
Fix #390

### CODE REVIEW

File: instructor/templates/instructor/assignment_sidebar.html [hunk 1]
```diff
@@ -21,27 +23,31 @@
...
+            {% if page_name == "variables" %}scb_s_sidebar_page_name_selected{% endif %}'>
             <a class='scb_s_assignment_sidebar_link'
                href="{% if not new %}{% url "common_assignments_variables" %}{% endif %}">
                 Select variables
             </a>
         </div>
 
-        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+        {% if page_name == "treatments" %}scb_s_sidebar_page_name_selected{% endif %}'>
```
Comment (noisecapella) [inline instructor/templates/instructor/assignment_sidebar.html]: It looks like this CSS class is defined starting with `scb_ab_s_...`. As it is I can't see the highlight on the panel

Comment (annagav) [inline instructor/templates/instructor/assignment_sidebar.html]: Fixed

**Comment by noisecapella** [comment]
Can you rebase?

File: instructor/templates/instructor/assignment_sidebar.html [hunk 2]
```diff
@@ -58,13 +65,15 @@
...
+        <div role="link" class='scb_ab_s_sidebar_page_link
+            {% if page_name == "wb_lysate_type" %}scb_ab_s_sidebar_page_name_selected{% endif %}'>
             <a class="scb_s_assignment_sidebar_link"
                href="{% if not new %}{% url "western_blot_lysate_type" %}{% endif %}">
                 Western Blot Lysate Type
             </a>
         </div>
-        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+            {% if page_name == "wb_antibodies" %}scb_ab_s_sidebar_page_name_selected{% endif %}'>
```
Comment (noisecapella) [inline instructor/templates/instructor/assignment_sidebar.html]: This should be `wb_antibody`

**Comment by noisecapella** [comment]
After you rebase you will also need to add highlighting for the wb_band_size link

**Comment by annagav** [comment]
Updated!

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**